### DISCUSSION TITLE
OS X Snow Leopard Unlocking X509Anchors

### DISCUSSION THREAD

**Discussion by Dal1ster** [atauenis/webone#154] [Q&A]
Im trying to set up webone on a Snow Leopard install, however when i try to put the root cert in the X509Anchors keychain, it asks for a password even though i dont have one set for any account on there, is there a way i can bypass this so i can access it and fully setup webone again?

**Comment by atauenis** [comment]
Enter empty password in that window. Simply.

**Comment by Dal1ster** [comment]
Odd, think I already did that to no avail but I'll try again

**Comment by Dal1ster** [comment]
Yeah i tried empty password in it and it said it was incorrect

**Comment by Dal1ster** [comment]
so what i had to do was follow a reply from this to forcefully unlock it via the terminal, then add the cert and lock it again https://discussions.apple.com/thread/2512513?sortBy=rank , leaving this here for anyone in the future who has this potential issue like i did.
